setup.py

```diff
@@ -60,6 +60,12 @@
         "llvmlite <= 0.33.0;python_version>='3.6'",
         "llvmlite <= 0.31.0;python_version<'3.6'",
     ],
+    "opentelemetry": [
+        "opentelemetry-api",
```

Comment:
Are there minimum / maximum versions we should set for these packages? If the package follows semantic versioning, we usually try to at least set a max version in case there are breaking changes in the future.

Reply:
So far I think any 0.x release may have breaking changes between versions as the API only started to stabilize somewhat lately. I believe most places use exact versioning due to this. Once they hit 0.9 or 1.0, I expect semantic versioning to kick in.

Reply:
Will add exact versioning to avoid this.